docs/api-docs/apps/guide/apps-04-devloper-portal.md

```diff
@@ -1,87 +1,77 @@
+7. You can view the client ID and client secret any time; see the following section on [viewing credentials](#view-credentials).
+8. To learn about deleting an app and its API account, see the [Guide to API Accounts](/api-docs/getting-started/authentication/rest-api-authentication#delete-apps-carefully)
 
-## Editing apps
+## View credentials
 
-Edit an app by clicking **Edit App**.
+To view an app API account's credentials, navigate to the Dev Portal's [My Apps dashboard](https://devtools.bigcommerce.com/my/apps) and click the **View Client ID** padlock icon to the right of the relevant app's listing. The dialog box that opens will reveal your API account credentials. To learn more about using an app API account to generate store-specific access tokens, see [Implementing the OAuth Flow](/api-docs/apps/guide/auth).
 
 ![Developer Portal](https://storage.googleapis.com/bigcommerce-production-dev-center/images/apps-04-developer-portal-01.png "Developer Portal")
 
-[Learn more about completing registration fields and submitting apps for approval](/api-docs/apps/guide/publishing).
-
-<!-- theme: info -->
-> #### Note
-> * After saving, edits are effective immediately in the control panel; however, edits can take up to 24 hours to appear in the [Apps Marketplace](https://www.bigcommerce.com/apps/).
+## Edit an app
 
+To edit an app, navigate to the Dev Portal's [My Apps dashboard](https://devtools.bigcommerce.com/my/apps) and click the **Edit App** pencil icon to the right of the app you want to edit.
 
+![Developer Portal](https://storage.googleapis.com/bigcommerce-production-dev-center/images/apps-04-developer-portal-01.png "Developer Portal")
 
-## Editing technical details
-
-Click **Edit App**, then navigate to the technical tab to edit enabled features, callback URLs, and OAuth scopes.
-
-![Technical Details](https://storage.googleapis.com/bigcommerce-production-dev-center/images/apps-04-developer-portal-04.png "Technical Details")
-
-[Learn more about completing registration fields and submitting apps for approval](/api-docs/apps/guide/publishing).
-
-<!-- theme: warning -->
-> #### Note
-> * After saving, edits to enabled features, callbacks URLs, and OAuth scopes are effective immediately for all app users.
+<!-- theme: info -->
+> #### Marketplace delay
+> Edits can take up to 24 hours to appear in the [App Marketplace](https://www.bigcommerce.com/apps/), but changes are immediately effective and visible in your existing users' store control panels.
 
+## Edit technical details
 
+In the Dev Portal's [My Apps dashboard](https://devtools.bigcommerce.com/my/apps), click the **Edit App** pencil icon to the right of the app you want to edit. At the top of the dialog that opens, click the **Technical** tab to edit enabled features, callback URLs, and OAuth scopes. To save your change, click the **Update & Close** button on the dialog's lower right. After saving, edits are effective immediately for all app users.

```

Comment:
```suggestion
In the Dev Portal's [My Apps dashboard](https://devtools.bigcommerce.com/my/apps), click the **Edit App** pencil icon to the right of the app you want to edit. At the top of the dialog that opens, click the **Technical** tab to edit enabled features, callback URLs, and OAuth scopes. Click the **Update & Close** button on the dialog's lower right to save your change. After saving, edits are effective immediately for all app users.

```

Reply:
decline. I want to express the consequence before the action.